modules/gcp_bigquery_dataset/outputs.tf

```diff
@@ -5,3 +5,7 @@ output "id" {
 output "self_link" {
   value = google_bigquery_dataset.google_bigquery_dataset.self_link
 }
+
+output "customer_key_id" {
```

Comment:
I don't think anybody knows what a 'customer key' is...I think you mean "customer-managed key"? https://cloud.google.com/storage/docs/encryption/customer-managed-keys

Reply:
Fixed to `customer_managed_key_id`